CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
+## [0.0.next] - 2017-06-02
+### Changed
+- java-heap-pcnt.sh and metrics-java-heap-graphite.sh: removed extra column from 'UsedHeap'
```

Comment:
The bug in here is a bit clearer than java versions. We're adding 5 buckets of memory used and getting a percentage from only 4.
I've got to look into this a bit more regarding the java versions. PermGen is part of Heap, but Metaspace is not a part of Heap. Ideally for older javas, I'd like to monitor PermGen separately to the rest of the Heap (just as you would for metaspace), does that sound reasonable?